src/etc/rc.bootup

```diff
@@ -89,6 +89,11 @@ if (file_exists("/root/firmware.tgz")) {
 
 /* Triggering of the initial setup wizard after reboot has been requested */
 if (file_exists("{$g['conf_path']}/trigger_initial_wizard_after_reboot")) {
+	/* Disable "Hardware Checksum Offloading" on first boot if QEMU VM system is detected */
+	if (isqemu()) {
+		$config['system']['disablechecksumoffloading'] = true;
+		write_config();
+	}
```

Comment:
Why store this in the config? Someone could restore a config without it and end up without the setting.

Seems like it would be better to test when the interface settings are applied https://github.com/pfsense/pfsense/blob/79e269c9f2002f780d2fd141595a1cec1683bb12/src/etc/inc/pfsense-utils.inc#L544

And is this necessary for any interface type in qemu, or only vtnet? If it isn't needed for em interfaces, for example, we probably shouldn't do it there.